### DISCUSSION TITLE
Get file from formdata for EXIF

### DISCUSSION THREAD

**Discussion by smartlightning** [rpldy/react-uploady#398] [Q&A]
Hello uploady community,
thanks for your amazing work on this useful library.
Currently I'm stuck trying to get the binary upload file in order to retrieve the EXIF data we need with exif js. 
Usually one would add an onChange to the file input. I saw useFileInput hook but it seems a bit of an overkill to only get the blob. Any ideas on how I can do this? 
Thanks in advance,
Chris

**Comment by yoavniran** [comment]
Hi @smartlightning 

You can easily access the File using the [useItemStartListener](https://react-uploady.org/docs/api/hooks/useItemStartListener/)


```jsx

 import { useItemStartListener } from "@rpldy/uploady";

    const MyComponent = () => {
        useItemStartListener((item) => {
            //access the File object through: item.file
        });
    };

```
